html/changelogs/geeves - slime_fix.yml

```diff
@@ -0,0 +1,8 @@
+author: Geeves
+
+delete-after: True
+
+changes: 
+  - tweak: "Slimecode has been tweaked a bit to make it slightly better. Adult slimes will no longer have their babyname next to their name when they talk."
+  - rscadd: "Golems have been added to the ghostspawner menu. You will be able to spawn as one when a golem rune is created."
+  - bugfix: "Slime steroids now properly triple uses, like the message says it does."
```

Comment:
This needs to be updated. Off the top of my head the nutrition changes are missing.